scripts/deploy/upgrade_v0.1.16.1/6_init_implementations.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+const BigNumber = require('bignumber.js');
+
+const MoC = artifacts.require('./MoC.sol');
+const MoCExchange = artifacts.require('./MoCExchange.sol');
+const StopperV2 = artifacts.require('./governance/StopperV2.sol');
+
+const { getConfig, getNetwork } = require('../helper');
+
+module.exports = async callback => {
+  try {
+    const network = getNetwork(process.argv);
+    const configPath = `${__dirname}/deployConfig-${network}.json`;
+    const config = getConfig(network, configPath);
+    const [owner] = await web3.eth.getAccounts();
```

Comment:
wouldn't be clearer to take to owner from moc? Like
`mocProxy -> stopper() -> owner()`